### ISSUE TITLE
Number of bands for band structure calculations and DOS

### ISSUE THREAD

**Issue by yuuukuma** [kumagai-group/vise#11]
It would be better if the number of bands for band structure calculations and DOS is changed depending on ISPIN

**Comment by FPrith** [comment]
Thank you very much for your indication.
I am a beginner at VASP and vise.
In the case of ISPIN=2 for collinear spin-polarized calculations (conventional ferromagnetism, etc.), what NBANDS values are appropriate to evaluate band structures and electronic DOS?
To my knowlege, the initial magnetic moments need to be considered, but I have no idea on what NBANDS values are enough large to evaluate them.

**Comment by yuuukuma** [comment]
Thank you for your comment. 
Typically, the number of bands required for spin-polarized calculations is not significantly different from that needed for non-spin-polarized calculations. 
However, it may be beneficial to increase the number of bands based on the spin magnetization. For instance, if the spin magnetization value is m=16 mu_B in the calculated cell, then increasing the number of bands by 16 would be more optimal.

**Comment by FPrith** [comment]
Thank you for your reply.
If antiferromagnetic rhombohedral NiO is treated (to my knowlege, the symmetry of cells with antiferromagnetism is not considered in vise, so the primitive unit cell of antiferromagnetic NiO needs to be created by other tools or by modifying its POSCAR file manually),
is there a need to increase the number of bands for band-structure calculations and DOS calculations?
I think that there is no need to do it because of its initial spin magnetization with net zero.

**Comment by yuuukuma** [comment]
Yes, you're right. So, please try to use the initial vise setting fist.

**Comment by FPrith** [comment]
Thank you very much for your reply.
At first, I will check whether the band-structure plots and the DOS plots of non-magnetic/ferromagnetic/antiferromagnetic cells change according to NBANDS or not!
